falcon/response.py

```diff
@@ -42,6 +42,10 @@
     'Please use Response.set_stream() or Response.content_length instead.'
 )
 
+_ADD_LINK_REMOVED_MSG = (
+    'The deprecated add_link() method was removed in Falcon 3.0. '
```

Comment:
This is simply not true. `add_link` was not removed in Falcon 3.0, just deprecated.

Reply:
On it.